Defs/Ammo/Shell/155mmHowitzer.xml

```diff
@@ -213,7 +213,6 @@
 		  <armorPenetrationSharp>0</armorPenetrationSharp>
 		  <armorPenetrationBlunt>0</armorPenetrationBlunt>
 		  <explosionRadius>10.5</explosionRadius>
-		  <flyOverhead>true</flyOverhead>
```

Comment:
This is not a direct-fire shell, why remove its flyOverhead node?

Reply:
Yeah... I noticed it yesterday. It shouldve been fixed in the recent commit.